MAINTAINERS.yml

```diff
@@ -4998,9 +4998,6 @@ West:
     - bperseghetti
   files:
     - modules/hal_nxp/
-    - modules/Kconfig.imx
-    - modules/Kconfig.mcux
-    - modules/Kconfig.nxp_s32
```

Comment:
okay, I am 100% positive that the commit with these deletions is breaking bisectibility

Reply:
This particular change probably doesn't break bisectability since this just drives tooling at the github level. But the other commits do need attention.

Reply:
Oh woops I think I commented on the wrong thing or misread something, yeah this is not relevant